scripts/decredPoliteia.md

```diff
@@ -1,25 +1,29 @@
-#Decred in Depth: Politeia
+# Decred in Depth: Politeia
 
 ### Keywords
 `proposal`, `Timestamped`, `versioned`, `self-governance`, `immutable`, `off-chain storage`, `git`, `chain-anchored`
 
-Named after the greek term meaning ‘a system of government’, Politeia is the Decred proposal system a social media application where Decred stakeholders submit proposals for projects to upgrade and market the Decred network. 
+Named after the greek term meaning ‘a system of government’, Politeia is the Decred proposal system: a chain-anchored social media application where Decred stakeholders can take part in deciding which Decred improvement projects recieve funding from the development fund. 
 
-Politeia is special in that it is designed to be an immutable and cryptographically verifiable public record of Decred’s governance-data. This means that once a proposal is posted to Politeia, the record of that proposal’s posting can never be altered or deleted. In addition, records of proposals cannot be forged. The time ordered and independently verifiable nature of Politeia ensures that attempts to manipulate Decred’s governance are significantly more difficult than if a traditional website or social media application were in use.
+Stakeholder submitted project proposals, proposal comments, and votes make up Decred's governance data hosted on Politeia. Newly submitted content is episodically anchored to the Decred blockchain creating an immutable and cryptographically verifiable public record of Decred's governance-data. Once a proposal is posted to Politeia, the record of it's posting can never be altered or deleted, nor can proposals be forged. 
```

Comment:
No need to be stakeholder (DCR holder) to submit a proposal, all you need is to pay the fee.

Reply:
"governance-data" -> "governance data"; "it's" -> "its"